amundsen_application/static/js/components/common/EditableSection/index.tsx

```diff
@@ -47,6 +47,10 @@ export class EditableSection extends React.Component<
     this.setState({ isEditing: !this.state.isEditing });
   };
 
+  stopPropagation = (event: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
Looks, great just a couple more things:
1. Let's call this `preventDeafult` since that's the name of the method it calls. 
2. Let's also add a unit test for this new functionality, to try to prevent the bug from happening again as this component gets updated. We will want to simulate a click event on the element, and verify that `prefentDefault` was called. Here's an example: https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/components/common/Bookmark/BookmarkIcon/index.spec.tsx#L39-L44